account_banking_pain_base/models/banking_export_pain.py

```diff
@@ -269,15 +269,18 @@ def generate_initiating_party_block(
                 context=context)
         initiating_party_issuer = gen_args['sepa_export'].\
             payment_order_ids[0].company_id.initiating_party_issuer
-        if initiating_party_identifier and initiating_party_issuer:
-            iniparty_id = etree.SubElement(initiating_party_1_8, 'Id')
-            iniparty_org_id = etree.SubElement(iniparty_id, 'OrgId')
-            iniparty_org_other = etree.SubElement(iniparty_org_id, 'Othr')
-            iniparty_org_other_id = etree.SubElement(iniparty_org_other, 'Id')
-            iniparty_org_other_id.text = initiating_party_identifier
-            iniparty_org_other_issuer = etree.SubElement(
-                iniparty_org_other, 'Issr')
-            iniparty_org_other_issuer.text = initiating_party_issuer
+        if not initiating_party_identifier or not initiating_party_issuer:
+            raise orm.except_orm(
+                _("Error"),
+                _("You must fill initiating party identifier and issuer."))
```

Comment:
Perhaps "You must fill initiating party identifier and issuer on the company form", so the user easiky understands knows how to fix the error.